### ISSUE TITLE
[1.12] Blocks & items changing into other Items

### ISSUE THREAD

**Issue by YOYOK9** [McJty/DeepResonance#226]
Not sure really how to describe this bug but when ever I craft certain machines or blocks they turn into thermal foundation materials once I exit my inventory. 

An example is the Dense Obsidian turns into a silver ingot...
All item changes: https://pastebin.com/2Zei3ZUm

MC: 1.12
Forge: 14.21.1.2443
DeepResonance: 1.12-1.4.9
ThermalFoundation: 1.12-2.3.5.14-universal

**Comment by McJty** [comment]
That doesn't sound like a bug in Deep Resonance but more in some other mod that influences crafting

**Comment by YOYOK9** [comment]
pulled the items right out of the creative menu. test crafted the obsidian in a minecraft crafting table.

**Comment by McJty** [comment]
Can you describe better what exactly you are doing?

**Comment by YOYOK9** [comment]
Sorry for the late response, went out. I am working on a modpack and was trying to make a harder crafting recipe in another mod (using crafttweaker) and was going to use blocks from your mod. But when I pulled them out of the creative menu they would convert into other items from thermal foundation or basic minecraft. 


The steps to reproduce this are:
Step 1: open D.R. creative tab.
Step 2: place one of the items in the list I had provided in your hot bar.
Step 3: close inventory. 

after that The item would convert into another item.
